apps/wing/src/commands/compile.ts

```diff
@@ -49,6 +49,11 @@ export interface CompileOptions {
    * @default "./target"
    */
   readonly targetDir?: string;
+  /**
+   * The overrides the location to save the compilation output
+   * @default "./target"
```

Comment:
The default is `./target/<entrypoint>.<target>` -- we should make sure this overrides the entire path and not just `./target`.

What I mean is that this should behave like this:

```sh
$ wing compile -t tf-aws main.w -o my-output
$ cd my-output
$ terraform init
$ terraform apply
```

Reply:
it's only the comment that was wrong :)